src/Libraries/Microsoft.Extensions.Diagnostics.ExceptionSummarization/ExceptionSummary.cs

```diff
@@ -13,7 +13,9 @@ namespace Microsoft.Extensions.Diagnostics.ExceptionSummarization;
 /// Metric tags typically support a limited number of distinct values, and as such they are not suitable
 /// to represent values which are highly variable, such as the result of <see cref="Exception.ToString"/>.
 /// An exception summary represents a low-cardinality version of an exception's information, suitable for such
-/// cases. The summary never includes sensitive information.
+/// cases.
+/// 
+/// The summary never includes privacy-sensitive information.
```

Comment:
I would remove that because it contradicts the documentation of its properties (`AdditionalDetails`).

Instead we should 
- repeat that users can use `Description` as it doesn't contain sensitive information
- repeat that `AdditionalDetails` might contain sensitive information depending on the implementation

We also have to add some text in `IExceptionSummaryProvider` to explain that `AdditionalDetails` should not leak any sensitive information as users might leak it inadvertently.

Reply:
Done.